plugins/http_plugin/http_plugin.cpp

```diff
@@ -766,15 +634,19 @@ class http_plugin_impl : public std::enable_shared_from_this<http_plugin_impl> {
              "Additionaly acceptable values for the \"Host\" header of incoming HTTP requests, can be specified multiple times.  Includes http/s_server_address by default.")
             ("http-threads", bpo::value<uint16_t>()->default_value( my->thread_pool_size ),
              "Number of worker threads in http thread pool")
+            ("http-use-beast", bpo::value<bool>()->default_value(true),
```

Comment:
I'm surprised to see this an option. Was leaving in the old impl a requirement?

Reply:
@spoonincode I left that in for testing purposes ... although once its well tested it should be removed

Reply:
Does that mean before the PR is merged? What would be the criteria for removing it?

Reply:
My plan was for sometime after it has been used enough to be sure there are no major issues related to the beast implementation vs. websocketpp.  The run-time switch makes it easier to isolate if a problem is caused by the beast implementation.    Once the version it is included in reaches stable (probably 2.2), it can be removed